<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,10,1">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 4);</string>
      </property>
      <property name="text">
       <string>商品价格</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5,2">
      <item>
       <widget class="QComboBox" name="cbb_method">
        <item>
         <property name="text">
          <string>全部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>名称</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>果蔬</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>食品</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>烟酒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>饮品</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>日用</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>粮油</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其他</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_text"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,2,1,2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="add_commodity">
        <property name="text">
         <string>添加商品</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>保存修改</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="text">
         <string>删除商品</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>积分管理</string>
    </property>
    <addaction name="btn_query_m"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="add_commodity1">
   <property name="text">
    <string>添加商品</string>
   </property>
  </action>
  <action name="btn_save2">
   <property name="text">
    <string>保存修改</string>
   </property>
  </action>
  <action name="btn_delet">
   <property name="text">
    <string>删除商品</string>
   </property>
  </action>
  <action name="btn_add_m">
   <property name="text">
    <string>添加成员</string>
   </property>
  </action>
  <action name="btn_query_m">
   <property name="text">
    <string>查询成员</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
